types/react-text-mask/index.d.ts

```diff
@@ -8,11 +8,11 @@
 
 import * as React from "react";
 
-export type maskArray = Array<string | RegExp> | boolean;
+export type maskArray = Array<string | RegExp> | false;
 
 export interface MaskedInputProps
     extends React.InputHTMLAttributes<HTMLInputElement> {
-    mask?: maskArray | ((value: string) => maskArray);
+    mask: maskArray | ((value: string) => maskArray);
```

Comment:
It would be nice to add tests showing that omitting the property or passing `true` results in an error.